governance/votes/Executive vote - April 1, 2022.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Template - [Executive Vote] Core Unit Budget Distributions and Vesting Streams Setup, Retroactive MKR Compensation for Long For Wisdom, Gelato Keeper Network DAI Budget Stream Setup - April 1, 2022
```

Comment:
LongForWisdom is all one word I'm fairly sure - see: https://mips.makerdao.com/mips/details/MIP41c4SP3